### PR TITLE
Document that all-in-one Jaeger is bundled with Istio

### PR DESCRIPTION
**Pull request by catherineluse**
https://github.com/rancher/docs/issues/2860

### CODE REVIEW

File: content/rancher/v2.x/en/istio/v2.5/_index.md [hunk 1]
```diff
@@ -44,6 +41,24 @@ Istio has migrated away from Helm as a way to install Istio and now provides ins
...
+
+Kiali is used for graphing traffic flow throughout the service mesh. It allows you to see how they are connected, including the traffic rates and latencies between them. You can check the health of the service mesh, or drill down to see the incoming and outgoing requests to a single component.
+
+### Jaeger
+
+_Bundled as of v2.5.4_
+
+Our Istio installer includes a quick-start, all-in-one installation of [Jaeger,](https://www.jaegertracing.io/) a tool used for tracing distributed systems.
+
+Note that this is not a production-qualified deployment of Jaeger. This deployment uses an in-memory storage component, while a persistent storage component is recommended for production. For more information on which deployment strategy you may need, refer to the [Jaeger documentation.](https://www.jaegertracing.io/docs/1.21/operator/#allinone-default-strategy)
```
Comment (brendarearden) [inline content/rancher/v2.x/en/istio/v2.5/_index.md:60]: `https://www.jaegertracing.io/docs/latest/operator/#production-strategy` can we use this url to ensure we don't need to keep updating it. This currently forwards to the 1.21 version.

**Review by brendarearden — CHANGES_REQUESTED** [review]
This is great! would just like to update the jaeger url so that we don't have to update it each release.

File: content/rancher/v2.x/en/istio/v2.5/_index.md (hunk 1 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/istio/v2.5/_index.md]: I replaced the link

**Review by brendarearden — APPROVED** [review]
thank you! looks like this covers everything!

**Pull request merged**